src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,19 +1,75 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
+# TO-DO: complete the helper function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
+    
     elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
+    merged_arr = [] * elements
+    
     # TO-DO
     
+    # Variables i and j equaling to 0
+    # Created to change index numbers
+    i = 0
+    j = 0
+    
+    # While i is less than the length of array A
+    # and while j is less than array B, the conditional will continue
+    while i < len(arrA) and j < len(arrB):
+        # If the index of array A is less than the index of array B...
```

Comment:
Keep in mind when you're talking about the element in the list arr[i] vs the index i